app/elements/bar-zoomable-chart/bar-zoomable-chart.html

```diff
@@ -0,0 +1,316 @@
+<link rel='import' href='../../bower_components/polymer/polymer.html'>
+<script src='../../bower_components/d3/d3.min.js' charset='utf-8'></script>
+<dom-module id='bar-zoomable-chart'>
+    <style>
+        :host {
+            display: block;            
+        }
+        #barChart {
+            width:100%;
+            height: 400px;
+        }
+        
+        .axis {
+          font-size: 11px;
+          fill: var(--bar-zoomable-chart-axis-color, --secondary-text-color);
```

Comment:
Is the --bar-zoomable-chart-axis-color variable defined anywhere?  If not, you can remove it.